### PR TITLE
ticket fixes, and wiki button

### PR DESCRIPTION
**Pull request by Amelia0010**

### CODE REVIEW

File: yogstation/code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -345,18 +345,36 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
...
+	var/msg = "<font color='red' size='4'><b>- AdminHelp marked as an IC issue! -</b></font><br>"
 	msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
 	msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>"
 
 	if(initiator)
 		to_chat(initiator, msg)
 
 	SSblackbox.record_feedback("tally", "ahelp_stats", 1, "IC")
-	msg = "Ticket [TicketHref("#[id]")] marked as IC by [key_name]"
+	msg = "Ticket [TicketHref("#[id]")] marked as an IC by [key_name]"
```
Comment (ThatLing) [inline yogstation/code/modules/admin/verbs/adminhelp.dm]: 🤔

Comment (Amelia0010) [reply yogstation/code/modules/admin/verbs/adminhelp.dm]: yes, it's marked as an IC issue. what's wrong with that?

Comment (AsV9) [reply yogstation/code/modules/admin/verbs/adminhelp.dm]: no this is bad grammar, it's `marked as an IC issue` or `marked as IC`, you can't have both mane

Comment (Amelia0010) [reply yogstation/code/modules/admin/verbs/adminhelp.dm]: I 👏 can 👏 not 👏 read

**Review by AsV9 — APPROVED** [review]

**Pull request merged**